packages/calypso-build/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# [Unreleased]
+
+- Added transform-runtime versioning to babel/default.js
+  This will need to be kept up to date while https://github.com/babel/babel/issues/10261 is unresolved.
```

Comment:
This comment would be more useful above the `version: '7.5.5'` spec in the Babel config source.

Reply:
Will add it there too, thanks!